ci_framework/roles/build_containers/molecule/hotfix/converge.yml

```diff
@@ -0,0 +1,28 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+- name: Converge
+  hosts: all
+  vars:
```

Comment:
In order to test hotfix role, we also need to pass
```
cifmw_build_containers_hotfix_images:  quay.io/podified-antelope-centos9/openstack-keystone:current-podified
```
We also need a rpm to test it.
We can add few tasks which run commands to download the rpm
1, sudo dnf install dnf-plugins-core
2. mdir rpms; cd rpms
2. sudo dnf download python3-requests --resolve

and then we can pass the rpms path to cifmw_build_containers_hotfix_rpms_paths